debian/rules

```diff
@@ -6,7 +6,8 @@ SYSCONFDIR = "/etc"
 DESTDIR = "debian/tmp"
 
 %:
-	dh $@ --with python2,systemd
+	# TODO : Make it work with python2 also
```

Comment:
Does this needs update?

Reply:
Addressed by David's commit @andrijapanicsb